cfgov/jinja2/v1/youth_employment_success/route-details.html

```diff
@@ -0,0 +1,72 @@
+<div class="content-l content-l_col-2-3 yes-route-details">
+    <p class="h4 u-mt0">
+      Totals for <span class="js-transportation-type"></span>
+    </p>
+
+    <div class="content_line-bold"></div>
+  
+    <div class="block__sub-micro">
+      <p class="content-l content-l_col-3-4">
+        <b>Money left in your monthly budget</b>
+      </p>
+      <p class="content-l content-l_col-1-4" style="text-align: right;">
```

Comment:
Avoid inline styling. Should this be a utility class or `yes-route-details_budget`?

Reply:
I had comment here that I was planning on adding actual styling, but I removed them since this is still WIP!